doc/INSTALLATION.md

```diff
@@ -27,7 +27,7 @@ properties:
 
 This helm chart provides three types of authentication: Single User, LDAP and OIDC.
 
-You can find how to configure these authentications on this [page](https://github.com/cetic/helm-nifi/tree/feature/nifi_1.14.0/doc/USERMANAGER.md).
+You can find how to configure these authentications on this [page](https://github.com/cetic/helm-nifi/tree/feature/nifi-1.16.3/doc/USERMANAGER.md).
```

Comment:
You might consider changing this to a relative link; see https://github.blog/2013-01-31-relative-links-in-markup-files/ and c.f. https://github.com/cetic/helm-nifi/blob/master/README.md?plain=1#L259

Reply:
@wknickless thank you, references have been updated.